source/transactions/tests/README.rst

```diff
@@ -382,3 +401,31 @@ instead.
             addresses.add(cursor.address)
 
           assert len(addresses) > 1
+
+Q & A
+=====
+
+Why do some tests appear to hang for 60 seconds on a sharded cluster?
+
+`````````````````````````````````````````````````````````````````````
+There are two cases where this can happen. When the initial commitTransaction
+attempt fails on mongos A and is retryed on mongos B, mongos B will block
+waiting for the transaction to complete. However because the initial commit
+attempt failed, the transaction will only completed after it is automatically
```

Comment:
Replace with "... the command will only complete after the transaction is automatically aborted ..."